src/components/Dropdown/index.tsx

```diff
@@ -28,20 +32,12 @@ const renderOptions = (options: OptionItem[]) => options.map(createOptions);
 const Dropdown: React.FC<DropdownProps> = (props) => {
 
   const { action, className, name, options, title } = props;
-  const [dropdownValue, setDropdownValue] = React.useState<string | number>();
+  const documentType = useSelector(state => state.createDoc[name]);
```

Comment:
What if an innocent developer tries to use the dropdown in a place that has nothing to do with the creational process of a document? The prop name would try to find out some value inside createDoc but maybe the action prop is meant to st a value inside another key on a reducer and we should associate dropdown with that value set instead.
It's early, im not that lucid yet, but maybe we can make the "action" prop and the "name" prop passed to work together. Maybe the name could be an array of properties that the component will loop to get the specific property of the redux store. 
`name = [ createDoc, name ]` something like this

Reply:
We already discussed this in a call, we decided to create a more specific component "DocTypeDropdown"